src/diagonal.works/b6/ingest/compact/encoding.go

```diff
@@ -235,7 +235,7 @@ func (n *Namespaces) ForType(t b6.FeatureType) Namespace {
 }
 
 const (
-	NamespacesLength = 8 // Verified in init()
+	NamespacesLength = 10 // Verified in init()
```

Comment:
this is b/c FeatureTypeEnd  = FeatureTypeInvalid number moved in world.go which i think is fine but lmk

Reply:
Erm. It is fine, however, making this change will invalidate all the indicies we currently have on disk. There's support for doing that in a controlled way (via the [Version constant field in encoding.go](https://github.com/diagonalworks/diagonal-b6/blob/5a6396359ca0e05b2242ef850029f968de06e30d/src/diagonal.works/b6/ingest/compact/encoding.go#L25)), but I don't think we want to trigger than until we actually start encoding collections to disk - so for the moment, I'd keep the endpoint as FeatureTypeRelation.

Reply:
aight